hvcc/generators/c2daisy/c2daisy.py

```diff
@@ -52,30 +52,61 @@ def compile(clazz, c_src_dir, out_dir, externs,
             env.loader = jinja2.FileSystemLoader(
                 os.path.join(os.path.dirname(os.path.abspath(__file__)), "templates"))
 
-            # generate Daisy wrapper from template
+            # # generate Daisy wrapper from template
+
+            # daisy_h_path = os.path.join(source_dir, f"HeavyDaisy_{patch_name}.hpp")
+            # with open(daisy_h_path, "w") as f:
+            #     f.write(env.get_template("HeavyDaisy.hpp").render(
+            #         name=patch_name,
+            #         board=board,
+            #         class_name=f"HeavyDaisy_{patch_name}",
+            #         num_input_channels=num_input_channels,
+            #         num_output_channels=num_output_channels,
+            #         receivers=receiver_list,
+            #         copyright=copyright_c))
+            # daisy_cpp_path = os.path.join(source_dir, f"HeavyDaisy_{patch_name}.cpp")
+            # with open(daisy_cpp_path, "w") as f:
+            #     f.write(env.get_template("HeavyDaisy.cpp").render(
+            #         name=patch_name,
+            #         board=board,
+            #         class_name=f"HeavyDaisy_{patch_name}",
+            #         num_input_channels=num_input_channels,
+            #         num_output_channels=num_output_channels,
+            #         receivers=receiver_list,
+            #         pool_sizes_kb=externs["memoryPoolSizesKb"],
+            #         copyright=copyright_c))
+
+            try:
+                targ = os.path.join(os.path.dirname(os.path.abspath(__file__)), "templates", f'{board}.json')
```

Comment:
My most recent changes would allow users to forgo the "board" field and simply supply a "description" that defines the boards just like the json in the repo does. So something like:
~~~json
{
  "daisy": {
    "description": {
      <board description here>
    }
  }
}
~~~

I see now it would probably have been simpler to do what you suggested (just having them supply a path to their json file) so that might be a good thing to change before this PR is ready.

Reply:
Either would be fine with me. But I can see that you may want to use the same board files elsewhere, so then it's better to use a path. You can use the `meta.daisy` file for supplying other hvcc specific settings you may want/need. But using the file-path to override any included files makes better organization then to fully include the board-description directly maybe.